inc/drivers/GigglebotMotors.h

```diff
@@ -0,0 +1,47 @@
+#ifndef GIGGLEBOT_MOTORS_H
+#define GIGGLEBOT_MOTORS_H
+
+#include "MicroBitComponent.h"
+#include "MicroBitI2C.h"
+
+#define GIGGLEBOT_MOTORS_EVT_UPDATE         1
+
+class GigglebotMotors
+{
+    MicroBitI2C i2c;
+
+    public:
+    GigglebotMotors(MicroBitI2C &_i2c);
```

Comment:
What if this took the motor number. Then main could construct a left motor, right motor, and both motors object from this and call setMotorPower on the specific one. setMotorPower in main could go away

Reply:
Good idea. And I'll change the class name to singular.

Reply:
✔️